html/register_credentials.html

```diff
@@ -21,7 +20,7 @@
         <label>
             Telefonummer:
             <br>
-            <input name="phoneNumber">
+            <input name="phoneNumber" type="number" id="phoneNumberInput">
```

Comment:
phoneNumber is not a `number`:
- leading 0s have meaning
- it can contain a `+`